SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.UWP/Microsoft.AppCenter.Analytics.UWP.csproj

```diff
@@ -11,11 +11,12 @@
     <AssemblyName>Microsoft.AppCenter.Analytics</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```

Comment:
Please revert changes in this file, this is not part of the version bump script and should not be touched during a release.

Reply:
reverted the changes